vNext/AISKU/package.json

```diff
@@ -43,6 +47,7 @@
         "@microsoft/applicationinsights-properties-js": "1.0.0-beta.11"
     },
     "peerDependencies": {
-        "tslib": "^1.9.3"
+        "tslib": "^1.9.3",
+        "rollup": "^0.66.0"
```

Comment:
This shouldn't be a peer dependecy, otherwise everyone will see an error when they try to install. I had problems with this being a devDep, so I just installed it to package.json during build then did `git checkout -- package.json` to remove it from deps after I finished building

Reply:
Moved to devDependencies. Build worked.